mlflow/transformers.py

```diff
@@ -1792,7 +1823,7 @@ def trim_input(data_in, data_out):
             ):
                 # If the user has indicated to not preserve the prompt input in the response,
                 # split the response output and trim the input prompt from the response.
-                data_out = data_out[len(data_in) :].lstrip()
+                data_out = data_out[len(data_in):].lstrip()
```

Comment:
```suggestion
                data_out = data_out[len(data_in) :].lstrip()
```

Can we revert this unrelated change?

Reply:
Sure - this one seems kind of minor though?

Reply:
Does black format this?

Reply:
Confirmed it does. Running black should revert this change.